debian/postinst

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -e
+
+if [ "$1" = "configure" ] && [ -n "$2" ]; then
+    for ver in $(phpquery -V); do
+	for sapi in $(phpquery -v "$ver" -S); do
+	    ini="/etc/php/$ver/$sapi/conf.d/20-snuffleupagus.ini"
```

Comment:
Why do you want to remove this symbolic link?

Reply:
Either it's a generated file, so we don't need it in the git, or it's something worth versioning, and we have complete control over it, so it shouldn't do anything silly.

Reply:
I did so many tests that I can't remember when this file was generated, nor how. I guess it was generated by dh-php when I was compiling with the 'php-' prefix.

Right now, if we remove this file it won't be generated again. So this code block won't be present in the postinst in the .deb file.

I don't know the consequences of removing this file.

Reply:
Please try and remove it if possible.

Reply:
I'm also not a big fan of the installing process removing files in `/etc`. This should only happen on a `apt purge`, not during an install.